provisioning/device/samples/register_symkey.js

```diff
@@ -7,10 +7,10 @@ var iotHubTransport = require('azure-iot-device-mqtt').Mqtt;
 var Client = require('azure-iot-device').Client;
 var Message = require('azure-iot-device').Message;
 
-// var ProvisioningTransport = require('azure-iot-provisioning-device-http').Http;
+var ProvisioningTransport = require('azure-iot-provisioning-device-http').Http;
```

Comment:
>Http [](start = 74, length = 4)

please keep the default as mqtt

Reply:
Yeah, don't know how that happened.  I sure thought I did.
Done.

---
In reply to: [282189198](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282189198) [](ancestors = 282189198)